src/components/PostList/PostList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
 import './PostList.scss';
+import { PostInfo } from '../PostInfo/PostInfo';
+import { PostListProps } from '../../types/PostListProps';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+export const PostList: React.FC<PostListProps> = ({ posts = [] }) => (
+  <ul className="post-list">
+    {posts.map(post => (
+      <li key={post.id} className="post-list__item">
+        <PostInfo post={post} />
+        {post.user && <UserInfo user={post.user} />}
+        <CommentList comments={post.comments} />
```

Comment:
I think UserInfo and CommentList should be included inside PostInfo component, and PostList is solely responsible for rendering a list of posts
```suggestion
```